LayoutTests/inspector/css/setStyleTextAfterSetStyleSheetText.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../http/tests/inspector/resources/protocol-test.js"></script>
+<link rel="stylesheet" href="resources/external.css">
+<script>
+function test() {
+    let suite = ProtocolTest.createAsyncSuite("CSS.setStyleTextRace");
```

Comment:
NIT: using a protocol test is really not ideal as there's so much more boilerplate.  can we not use a regular `InspectorTest` for this instead?

Reply:
The only part of Web Inspector that calls `CSS.setStyleSheetText` is inside an event handler. That requires even more needless boilerplate (`WI.SourceCodeRevision`).

https://github.com/WebKit/WebKit/blob/0fdbbfbe25ea91a8952f14f72cc8175183aeaa72/Source/WebInspectorUI/UserInterface/Models/CSSStyleSheet.js#L171-L192

Ultimately, this test exercises the exact sequence of calls to the backend which cause the issue the patch fixes. 
The boilerplate isn't great, but it's relatively contained.